toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss

```diff
@@ -105,6 +105,64 @@
 		font-weight: bold;
 	}
 
+	&__author-modal {
```

Comment:
Are there any values in here that are repeated and therefore should live in 10-settings?

Reply:
Yep, will do